applications/nrf_desktop/sample.yaml

```diff
@@ -18,13 +17,19 @@ common:
       - "(ble_adv: Advertising started)|(ble_scan: Scan started)"
       - "dfu: Secondary image slot is clean"
 tests:
+  applications.nrf_desktop.zdebug.uart.kmu_provision:
```

Comment:
https://docs.zephyrproject.org/latest/develop/test/twister.html

look for pytest_root,
it can be set in harness_config section, by default it is `pytest` dir, where there is now file test_mcuboot_kmu.py

Reply:
thanks, what would happen if we defined another test in the `pytest` dir? Would both tests be run for the `applications.nrf_desktop.zdebug.uart.kmu_provision` test configuration?

Reply:
I think so. Anyway I think there is no need to add more pytests at this point